src/Symfony/Bridge/PhpUnit/Legacy/CoverageListenerTrait.php

```diff
@@ -107,6 +107,11 @@ private function addCoversForDocBlockInsideRegistry($test, $covers)
         $symbolAnnotations = new \ReflectionProperty($docBlock, 'symbolAnnotations');
         $symbolAnnotations->setAccessible(true);
 
+        // Exclude non-namespaced classes. PHPUnit 9.1+ is picky about test covering, say, a \RuntimeException.
```

Comment:
this should exclude internal classes specifically if possible, not all non-namespaced classes should be excluded

Reply:
Fair enough. Should now check for user defined-ness specifically.